### PR TITLE
Updates to Query Tool errors

### PR DESCRIPTION
**Pull request by dborgen2**
It was decided that if there were multiple errors with a certain field, we would show all of the errors instead of just the first one. In addition, the wording for when the user doesn't type in any letters is confusing, as it said "Normalized Last Name must be at least 1 character long." Now, we'll just say "Last Name must start with a letter" and update the requirements for that field appropriately.

## What’s changing?

- The UsaSSN attribute validation now returns all errors instead of just the first one
- The UsaName attribute now just simply checks to make sure the first non-space character they've typed is a letter
- The alert box was updated to show all errors on multiple list items when the returned error has more than one line
- The error above the field on a form was updated to show all errors instead of just the first one. Line breaks separate the errors.
- The SSN input field was widened by 10 more pixels so that the last digit wouldn't get cut off when the field had an error.

## Why?

Closes NAC-655

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: components/tests/Piipan.Components.Tests/Validation/UsaNameAttributeTests.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void CorrectErrorMessageForNormalizedNameTooShort()
 
             // Assert
             Assert.False(result);
-            Assert.Equal(ValidationConstants.NormalizedNameTooShortMessage, attribute.ErrorMessage);
+            Assert.Equal(ValidationConstants.MustStartWithALetter, attribute.ErrorMessage);
```
Comment (bgfoshay) [inline components/tests/Piipan.Components.Tests/Validation/UsaNameAttributeTests.cs:49]: Question/Nitpick: Should the test name change as well?

Comment (dborgen2) [reply components/tests/Piipan.Components.Tests/Validation/UsaNameAttributeTests.cs:49]: Yep, good point. Just updated

File: components/src/Piipan.Components/Validation/ValidationConstants.cs [hunk 2]
```diff
@@ -40,7 +40,7 @@ public class ValidationConstants
         /// <summary>
         /// Used for names when validating they would have at least one character after normalization
         /// </summary>
-        public const string NormalizedNameTooShortMessage = "Normalized @@@ must be at least 1 character long.";
+        public const string MustStartWithALetter = "@@@ must start with a letter";
```
Comment (bgfoshay) [inline components/src/Piipan.Components/Validation/ValidationConstants.cs:43]: Nitpick: You might consider pulling the placeholder/token out into it's own constant that can be reused, for example, in calls like @splitMessage.Replace("@@@", Label)

**Review by bgfoshay — APPROVED** [review]
I had one other nitpick you could consider but otherwise I think this looks good.

File: components/src/Piipan.Components/Validation/ValidationConstants.cs (hunk 2 above)
Comment (dborgen2) [reply components/src/Piipan.Components/Validation/ValidationConstants.cs]: Thanks, you're definitely right. Just updated the project to use a constant.

**Pull request merged**